.github/CODEOWNERS

```diff
@@ -172,6 +172,7 @@
 /devenv/bulk-dashboards/ @grafana/dashboards-squad
 /devenv/bulk-folders/ @grafana/grafana-frontend-platform
 /devenv/create_docker_compose.sh @grafana/backend-platform
+devenv/alert_rules.yaml @grafana/alerting-backend-product
```

Comment:
I think there is an initial `/` missed, no?

Reply:
Good catch! I just added it.